jobs/transforms/base_model_audits.py

```diff
@@ -155,3 +155,24 @@ def process(self, input_model):
         current_datetime = datetime.datetime.utcnow()
         if (model.last_updated - MAX_CLOCK_SKEW_SECS) > current_datetime:
             yield audit_errors.ModelMutatedDuringJobError(model)
+
+
+@audit_decorators.AuditsExisting(base_models.BaseCommitLogEntryModel)
```

Comment:
base_model.BaseSnapshotMetadataModel should be probably added too?

Reply:
Done